### ISSUE TITLE
Import Subscriptions from OPML

### ISSUE THREAD

**Issue by ispringle** [djt3/tuitube#12]
Youtube provides a method of exporting a playlist to an OPML file. You can then use this to import them into various applications including RSS aggregators and Newspipe. Wondering if this feature is already implemented for tuitube and I'm not seeing it or if this is a new suggestion that could be implemented. If it's a new feature, I can provide a OPML file to use if you'd like. I don't have any experience with C++ or else I'd give it a stab myself, but I am familiar with the OPML format and might be able to help with parsing it.

**Comment by djt3** [comment]
hi, thanks for the suggestion, this sounds like a great feature for people who have youtube accounts and want to keep their subscriptions. ill have a look into it and keep you updated.

**Comment by ispringle** [comment]
Yes, I'm looking to get rid of my Youtube account but would like to hold onto my subs, so that's my use case. Here's a sample of the OPML that they give you

```
<opml version="1.1">
  <body>
    <outline text="YouTube Subscriptions" title="YouTube Subscriptions">
      <outline text="Code Bullet" title="Code Bullet" type="rss" xmlUrl="https://www.youtube.com/feeds/videos.xml?channel_id=UC0e3QhIYukixgh5VVpKHH9Q" />
    </outline>
  </body>
</opml>
```

I prettied it up, it actually came without any newlines, but I shouldn't think that would matter to a parser.
